entities/helper.py

```diff
@@ -0,0 +1,26 @@
+from typing import Any, Iterable
+
+from common import indent
+
+
+def indent_entity(entity: Any) -> str:
```

Comment:
Don't we want to add the indent level parameter too?

Reply:
Since we only use 1 indent level at a time there isn't really a point in this.